s/frontend/components/prepare-components.ts

```diff
@@ -1,6 +1,10 @@
+import { EditReview } from './reviews/edit-review/index.js';
```

Comment:
syntax -- prefer not to use spaces around imports, prefer to use double-quotemarks

also, no semicolons in this project

Reply:
Done